### PR TITLE
fix(ui): Clean up disabled reason spacing

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Not sure why it had the margin removed, but this looks good. Added a story book item.

### CODE REVIEW

File: docs-ui/components/form.stories.js [hunk 1]
```diff
@@ -106,12 +106,19 @@ storiesOf('Forms/Fields/New', module)
           placeholder="Type here to show the return button"
           showReturnButton
         />
-        <TextField
+        <textfield
```
Comment (billyvg) [inline docs-ui/components/form.stories.js]: Casing

Comment (EvanPurkhiser) [reply docs-ui/components/form.stories.js]: Oh weird, I saw these but didn’t look close enough at why they changed and assumed white space trimming

**Comment by EvanPurkhiser** [comment]
I'm a little concerned with this comment https://github.com/getsentry/sentry/blob/4db60cc6c04cf6d709c8ab7f9a18bebec42ecc01/src/sentry/static/sentry/app/views/settings/components/forms/field/fieldControl.jsx#L10-L12

**Comment by billyvg** [comment]
I think that's just referring to `FieldControlErrorWrapper` itself.

**Comment by EvanPurkhiser** [comment]
This is good to go. [here's the storybook ](https://storybook.getsentry.net/branches/fixui-clean-up-disabled-reason-spacing/index.html?selectedKind=Forms%2FFields%2FNew&selectedStory=TextField&full=0&down=1&left=1&panelRight=0&downPanel=storybooks%2Fstorybook-addon-knobs)

**Review by billyvg — APPROVED** [review]
Nice, thanks for the story.

Not requesting it for this PR, but what I meant by play are the "knobs", you can declare props with a knob type helper, and it would allow you to live change props inside of storybook.

**Pull request merged**